config/webpack.config.js

```diff
@@ -135,7 +137,7 @@ var exportBuild = {
 var webpackConf = {
     entry: './src/tau.charts.js',
     output: {
-        library: 'tauCharts',
+        library: 'taucharts',
```

Comment:
`tauCharts` library name should be left here, because we need camel cased `window.tauCharts`. This name will not be included into `commonjs2` and `amd` definitions, but for `this` only (for some reason `commonjs` is also there, but is ignored when doing import in Webpack or Rollup).